Gemfile.lock

```diff
@@ -1,29 +1,26 @@
 GIT
   remote: git://github.com/LightGuard/awestruct.git
-  revision: 05bf2c7fc4027c00115b6bc2e9ff0ffcf3eacc8e
+  revision: 5dbe512680b7aa094598ecdcbb6e52d3e75aa32f
```

Comment:
Why are you using my fork?

On Tuesday, July 14, 2015, Paul Robinson <notifications@github.com> wrote:

> In Gemfile.lock
> <https://github.com/jboss-developer/www.jboss.org/pull/920#discussion_r34546615>
> :
>
> > @@ -1,29 +1,26 @@
> >  GIT
> >    remote: git://github.com/LightGuard/awestruct.git
> > -  revision: 05bf2c7fc4027c00115b6bc2e9ff0ffcf3eacc8e
> > +  revision: 5dbe512680b7aa094598ecdcbb6e52d3e75aa32f
>
> You need to run bundle install and then add the new Gemfile.lock.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jboss-developer/www.jboss.org/pull/920/files#r34546615>
> .
>


-- 
Sent from Gmail Mobile